konan/konan.properties

```diff
@@ -727,4 +727,4 @@ runtimeDefinitions.wasm32 = KONAN_WASM=1 KONAN_NO_FFI=1 KONAN_NO_THREADS=1 \
   KONAN_INTERNAL_NOW=1 KONAN_NO_MEMMEM KONAN_NO_CTORS_SECTION=1
 
 # The version of Kotlin/Native compiler
-compilerVersion=@compilerVersion@
\ No newline at end of file
+compilerVersion=@compilerVersion@
```

Comment:
It's beter to avoid changing files by adding new lines without actual changes to content of the file.

Reply:
Yeah, I agree. This happened because I have IntelliJ set to automatically add a newline to EOF when saving files, and I did test modifications to konan.properties, so it saved with a linefeed at EOF. I can revert that.